src/main/java/seedu/ddd/model/event/common/UniqueEventList.java

```diff
@@ -61,7 +61,7 @@ public void setEvent(Event target, Event editedEvent) {
         CollectionUtil.requireAllNonNull(target, editedEvent);
 
         int index = internalList.indexOf(target);
-        if (index == -1) {
+        if (index < 0) {
```

Comment:
This should be left as `index == -1` because [`indexOf`](https://docs.oracle.com/javase/8/docs/api/?java/util/List.html) can only return -1 if the object is not found.

Reply:
Oops ok got it